applications/ParticleMechanicsApplication/custom_strategies/schemes/MPM_residual_based_bossak_scheme.hpp

```diff
@@ -1,8 +1,13 @@
+//    |  /           |
```

Comment:
Can you derive from core Bossak instead of creting a new one with your particular settings?

Reply:
They are actually a bit different, I need to reset nodal value at each time step and also extrapolate and interpolate the value from/to MP. I do agree that it can be derived, but I will work on it later.